avm/ptn/azd/aks/main.bicep

```
@@ -0,0 +1,466 @@
+metadata name = 'avm/ptn/azd/aks'
```

This will be used as the readme title. I'd suggest a more user friendly short name, possibly starting with `Azd`, like e.g.
```suggestion
metadata name = 'Azd AKS'
```

Please note that the readme would require an update after that

---

I have updated by your comments.